### PR TITLE
updates for one build

### PR DESCRIPTION
**Pull request by jennyf19**
Ensures that the same build variable name (`MicrosoftIdentityModelVersion`) is used both for the version of the library itself, and in the libraries that take a dependency on Wilson. This name was already used in other libraries. This becomes the way to set the version in Release builds.

### CODE REVIEW

File: build/version.props [hunk 1]
```diff
@@ -1,17 +1,22 @@
 <!-- This file may be overwritten by automation. Only values allowed here are VersionPrefix and VersionSuffix.  -->
 <Project>
```
Comment (keegan-caruso) [inline build/version.props:2]: Can we delete this file?

---

Or do we want it just for the one prop?

Comment (jennyf19) [reply build/version.props:2]: good catch, I had meant to delete it. it's deleted now.

File: Wilson.sln [hunk 2]
```diff
@@ -105,7 +105,6 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "build", "build", "{BC99A01F
 		build\dependenciesTest.props = build\dependenciesTest.props
 		build\targets.props = build\targets.props
 		build\targetsTest.props = build\targetsTest.props
-		build\version.props = build\version.props
```
Comment (keegan-caruso) [inline Wilson.sln]: why remove from the sln?

**Review by westin-m — APPROVED** [review]

File: Wilson.sln (hunk 2 above)
Comment (jennyf19) [reply Wilson.sln]: It's deleted

Comment (kllysng) [reply Wilson.sln]: I'm sorry, I don't see version.props as deleted

File: build/version.props [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
-    <PropertyGroup Condition=" '$(VersionSuffix)' == '' ">
-      <Version>$(VersionPrefix)</Version>
-    </PropertyGroup>
-
-    <PropertyGroup Condition=" '$(VersionSuffix)' != '' ">
-      <Version>$(VersionPrefix)-$(VersionSuffix)</Version>
-    </PropertyGroup>
-
-</Project>
+    <PreviewVersionSuffix Condition="'$(PreviewVersionSuffix)' == '' and '$(BuildingInsideVisualStudio)' != 'true'">preview-$([System.DateTime]::Now.AddYears(-2019).Year)$([System.DateTime]::Now.ToString("MMddHHmmss"))</PreviewVersionSuffix>
```
Comment (JoshLozensky) [inline build/version.props:7]: Why did we want to add seconds here?

**Review by JoshLozensky — APPROVED** [review]

File: Wilson.sln (hunk 2 above)
Comment (jmprieur) [reply Wilson.sln]: @jennyf19, I had readded during the WE it as it's also used by testscommons. But it's now different.
I've rea-added it to the solution too (right now)

**Review by jmprieur — APPROVED** [review]

**Pull request merged**